.github/ISSUE_TEMPLATE/enhancement_request.yml

```diff
@@ -11,10 +9,11 @@ body:
   - type: checkboxes
     attributes:
       label: Is there an existing issue for this?
-      description: Please search to see if an issue already exists for the bug you encountered.
+      description: >-
+        Please search the reporitory to see if if this enhancement was already requested. If it does add your perspective to its comments instead of creating a new one. Also make sure, that you've added any relevant labels to this issue. 
```

Comment:
```suggestion
        Please search the repository to see if if this enhancement was already requested. If it does, add your perspective to its comments instead of creating a new one. Also make sure, that you've added any relevant labels to this issue. 
```